example-workflows/Technique_3_Slack_Incoming_Webhook/README.md

```diff
@@ -0,0 +1,57 @@
+# Technique 3: Slack incoming webhook
+
+This technique uses this Action to post a message to a channel or direct message
+with [incoming webhooks][incoming-webhook] and a Slack app.
+
+Incoming webhooks follow the same [formatting][formatting] patterns as other
+Slack messaging APIs. Posted messages can be as short as a single line of text,
+include additional interactivity with [interactive components][interactivity],
+or be formatted with [Block Kit][block-kit] to build visual components.
+
+## Setup
+
+For details on how to setup this technique in GitHub Actions, read the
+[`README.md`][setup].
+
+## Example workflows
+
+1. [Post an inline text message](#post-an-inline-text-message)
+2. [Post an inline block message](#post-an-inline-block-message)
+3. [Post blocks found in a file](#post-blocks-found-in-a-file)
+
+### Post an inline text message
+
+Write a single world after a push event is received.
```

Comment:
```suggestion
Write a single word after a push event is received.
```

Reply:
although it isn't just one word? that would be a weird limitation maybe?

Reply:
Thanks for calling this out!! I forget if I intended to actually write a single word or not, but that's not what the example is showing anyways 😭

Going to update this to something like "a single line of text"!